src/nvim/main.c

```diff
@@ -454,6 +454,7 @@ int main(int argc, char **argv)
   }
 
   setmouse();  // may start using the mouse
+  ui_flush();
```

Comment:
did this fix a problem you were able to reproduce? Would be good to mention it in https://github.com/neovim/neovim/commit/bc55bf76c36f29729f9111f6a510e7112f729cdb .

We already use title-stacking.

Reply:
No. I looked at `resettitle` to replicate what Vim did but it's meant for `buffer.c`. I don't see any difference in the TUI.

Reply:
Let's skip this change if we aren't sure it's needed.